### PR TITLE
Removing all SG 2.0 version references

### PR DESCRIPTION
**Pull request by cdwinsnes**
…as there is currently only 1 version available. Also removed the v1 -> v2 migration guide

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/i1QmXFP6](https://trello.com/c/i1QmXFP6)

File: content/api/shipping-guide_2/topics.html [hunk 1]
```diff
@@ -97,10 +97,16 @@ <h2 class="dev-anchored">Should I handle warnings?</h2>
   <p>
     Ideally clients should not parse these warning codes for any logic as they
     are bound to change. These are supposed to be treated as informational data.
-    More info can be found in the section
-    <a href="/api/shipping-guide_2/upgrade-to-version-2/#introduced-warnings"
-      >Introduced warnings</a
-    >.
+    Warnings are send out when Shipping guide is able to validate the
```
Comment (tomgag) [inline content/api/shipping-guide_2/topics.html:100]: send -> sent

**Review by tomgag — APPROVED** [review]
LGTM

**Review by torstein-a — APPROVED** [review]
lgtm 👍

**Pull request merged**